pkgs/by-name/an/angle/package.nix

```
@@ -0,0 +1,152 @@
+{
+  stdenv,
+  gn,
+  ninja,
+  llvmPackages_21,
+  gclient2nix,
+  pkg-config,
+  glib,
+  python3,
+  symlinkJoin,
+  lib,
+  xorg,
+  wayland,
+  pciutils,
+  libGL,
+  ...
```

```suggestion
  libGL,
```

`...` shouldn't be necessary, should only be receiving the relevant args.